web/pages/index.tsx

```diff
@@ -23,26 +23,46 @@ import {
   Center,
 } from "@chakra-ui/react";
 import { auth } from "../utils/firebase/firebaseClient";
-import React from "react";
+import React, { useState } from "react";
 import { useAuth } from "../context/auth";
+import { mongo } from "mongoose";
 import { QuestionOutlineIcon } from "@chakra-ui/icons";
+import { TRPCClientError } from "@trpc/client";
+import { trpc } from "../utils/trpc";
+import { Role } from "../utils/types/account";
+import { findAccount } from "../db/actions/Account";
+import Account from "../db/models/Account";
+import { string } from "zod";
 
 export default function Home() {
+  const [authorized, setAuthorized] = useState(false);
   const { user, loading } = useAuth();
+  const [userInfo, setUserInfo] = useState({ email: "", uid: "", name: "" });
```

Comment:
Use given `useAuth` hook, it has a `userData` field that pulls from the User document if it is found